docs/upgrade.html

```diff
@@ -49,8 +49,9 @@ <h5><a id="upgrade_300_notable" href="#upgrade_300_notable">Notable changes in 3
             were removed. These methods were not intended to be public API and there is no replacement.</li>
         <li>The <code>NoOffsetForPartitionException.partition()</code> method was removed. Please use <code>partitions()</code>
             instead.</li>
-        <li>The Scala <code>kafka.common.MessageFormatter</code> was removed. Plese use the Java <code>org.apache.kafka.common.MessageFormatter</code>.</li>
+        <li>The Scala <code>kafka.common.MessageFormatter</code> was removed. Please use the Java <code>org.apache.kafka.common.MessageFormatter</code>.</li>
         <li>The <code>MessageFormatter.init(Properties)</code> method was removed. Please use <code>configure(Map)</code> instead.</li>
+        <li>The <code>PartitionAssignor</code> classes have been removed. Please use <code>ConsumerPartitionAssignor</code> instead.</li>
```

Comment:
```suggestion
        <li>The deprecated <code>o.a.k.clients.consumer.internals.PartitionAssignor</code> class has been removed. Please use <code>ConsumerPartitionAssignor</code> instead.</li>
```

Reply:
I reworded this and added a "and related classes" since we also removed `PartitionAssignorAdaptor`, WDYT?

Reply:
I changed it to the singular `class` intentionally since the `PartitionAssignorAdaptor` was never a public API, the only thing that was exposed to users -- and therefore the only thing they care or need to know about -- was the `PartitionAssignor`. Does that make sense?